Public.php

```diff
@@ -6,13 +6,14 @@
 if (!function_exists('liked_posts_render_blade_view')) {
     function liked_posts_render_blade_view($view, $data = [], $compress = true)
     {
+        $markup = '';
         $componentLibrary = new ComponentLibraryInit([]);
         $bladeEngine = $componentLibrary->getEngine();
-        $viewPath = MODULARITYLIKEPOSTS_VIEW_PATH;
+        $viewPaths = [MODULARITYLIKEPOSTS_MODULE_VIEW_PATH, MODULARITYLIKEPOSTS_VIEW_PATH];
         $data = array_merge($data, array('errorMessage' => false));
 
         try {
-            $markup = $bladeEngine->makeView($view, $data, [], $viewPath)->render();
+            $markup = $bladeEngine->makeView($view, $data, [], $viewPaths)->render();
         } catch (\Throwable $e) {
             $markup .= '<pre style="border: 3px solid #f00; padding: 10px;">';
```

Comment:
Use $bladeEngine->errorHandler($e) here, this will avoid having to handle the error output and minimizes the amount of code required.